source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,320 @@ Sorts Builders
+**sort criteria** for your queries.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance.
+The builder pattern is an Object Oriented design technique for creating complex
+immutable objects separately from their representation. The builders provided
+by the MongoDB Java driver can improve the readability of your code.
+
+You should read this guide if you would like to use the builder pattern
+to specify sort criteria for your queries. If you are new to sorting in the
+MongoDB Java driver, consider reading our
+:doc:`CRUD Sort Guide </fundamentals/crud/read-operations/sort/>`
+for a general introduction to sorting with the MongoDB Java driver.
+
+The Sorts Class
+---------------
+
+The :java-docs:`Sorts <apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+class is the sort criteria builder provided by the MongoDB Java Driver.
+``Sorts`` provides static factory methods for all
+sort criteria operators supported by MongoDB. Each method returns
+an instance of the
+:java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html?>`
+type. 
+
+The following examples show you how to use the static factory
+methods provided by the ``Sorts`` class to sort your queries.
+The examples use a sample collection ``sort_example`` that contains
+the following documents:
+
+.. code-block:: none
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Sorting Direction
+-----------------
+
+The ``Sorts`` class provides two methods for specifying the sort order of data
+that does not contain any duplicate values. Data can be sorted from smallest to largest 
+with ``Sorts.ascending()``, and data can be sorted from largest to smallest with
+``Sorts.descending()``. The following two subsections provide examples
+on how to use these two methods. 
+
+Ascending
+~~~~~~~~~
+
+To specify an ascending sort, use the ``Sorts.ascending()``
+static factory method. An ascending sort returns 
+your documents from smallest to largest on the field name specified.
+
+The ``ascending`` method can be used as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts
+
+   // <MongoCollection setup code here>
+
+   collection.find().sort(Sorts.ascending("<field name>"))
+
+This will return a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`
+object containing the documents in your collection sorted 
```

Comment:
```suggestion
object containing the documents in your collection, sorted 
```

Reply:
Thanks for the note. Fixed: https://github.com/mongodb/docs-java/pull/59/commits/faebe4d9b87f28abee25e3a3ffd3f049b978faa0

Going forward, I will try to get a more formalized grammar checking procedure in place.